tests/common/test_transaction.py

```diff
@@ -481,6 +475,15 @@ def test_asset_link_deserialization_with_empty_payload():
     assert asset_link == expected
 
 
+def test_asset_link_from_inputs(utx, transfer_utx):
+    from bigchaindb.common.transaction import AssetLink
+
+    expected = AssetLink(utx.id)
+    asset_link = AssetLink.from_inputs(utx, transfer_utx)
```

Comment:
The API here looks a bit odd to me. It's not clear that you're supposed to pass in all your transactions so that it can validate that they have the same asset id.